GestionVeterinarias/Veterinarias/LogicaVeterinarias/Classes/Mascota.cs

```diff
@@ -4,9 +4,28 @@
 using System.Text;
 using System.Threading.Tasks;
 
+
 namespace LogicaVeterinarias.Classes
 {
     class Mascota
     {
+        public int Id { get; set; }
+        public TipoAnimal TipoAnimal { get; set; }
+        public string Raza { get; set; }
+        public int Edad { get; set; }
+        public bool VacunasAlDia { get; set; }
+        public CarnetInscripcion CarnetInscripcion { get; set; }
+
+        public Mascota() { }
+
+        public Mascota(TipoAnimal tipo, string raza, int edad, bool vacunasAlDia, CarnetInscripcion carnet) 
+        {
+            this.TipoAnimal = tipo;
```

Comment:
como hablamos queda como lo tenias
public Mascota(int id, TipoAnimal tipo, string raza, int edad, bool vacunasAlDia, CarnetInscripcion carnet) 
        {
        this.id = id